evalml/tests/dependency_update_check/latest_dependency_versions.txt

```diff
@@ -12,7 +12,7 @@ networkx==2.5
 nlp-primitives==1.1.0
 numpy==1.19.5
 pandas==1.1.5
-plotly==4.14.2
+plotly==4.14.3
```

Comment:
Lol plotly [has done two releases in two days](https://pypi.org/project/plotly/#history)!